packages/front-end/components/SyntaxHighlighting/Snippets/GrowthBookSetupCodeSnippet.tsx

```diff
@@ -148,8 +149,7 @@ app.use(function(req, res, next) {
             encryptionKey
               ? `\n    decryptionKey: ${JSON.stringify(encryptionKey)}`
```

Comment:
This is missing a comma at the end and the newline missing below messes up the formatting